locales/en.yml

```diff
@@ -283,3 +283,16 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+
+EventDashboard:
+  main: Only event owner can access this area.
+  buttonUnlink: Unlink
+  Couple: Couple
+  Button:
```

Comment:
You can dissolve `Button` and just put all keys one level higher.

Reply:
> You can dissolve `Button` and just put all keys one level higher.
K